pycbc/waveform/.ipynb_checkpoints/Untitled-checkpoint.ipynb

```diff
@@ -0,0 +1,6 @@
+{
+ "cells": [],
+ "metadata": {},
+ "nbformat": 4,
```

Comment:
Be careful not to commit files you don't intend to.

Reply:
If we don't change the generate.py script (keeping apply_fd_time_shift in the FDomainDetFrameTwoPolNoRespGenerator) and want to get the time domain waveform using a time domain approximant 'SEOBNRv4', then it will give the plot below. 
```
params={'mass1' : 36.09431175153813,
'mass2' : 33.61286267925508,
'distance' :  500.60060060060056} 
from pycbc.waveform.generator import FDomainDetFrameTwoPolNoRespGenerator
from pycbc.waveform.generator import TDomainCBCGenerator
generator = FDomainDetFrameTwoPolNoRespGenerator(TDomainCBCGenerator, 0, variable_args=['mass1', 'mass2', 'distance'],  delta_f=1./68, delta_t=1/2048, f_lower=20., approximant='SEOBNRv4')
no_det = generator.generate(**params) #frequency domain waveform
print(no_det['RF'])
plt.plot(no_det['RF'][0])
plt.show()
``` 
![add_fd_frequency](https://github.com/user-attachments/assets/c0c0bd43-0393-4b85-8da9-790cae945c87)

```
plt.plot(no_det['RF'][0].to_timeseries())
plt.show()
```
![add_fd_timedomain](https://github.com/user-attachments/assets/bf05af6f-5746-4c61-b2b1-a14cdb5892db)
If we notice the time domain plot then it is clear that merger is not shifted at the beginning of the waveform or the merger is not at the boundary.

But if we do the required changes (changing apply_fd_time_shift to  apply_fseries_time_shift in the FDomainDetFrameTwoPolNoRespGenerator) then it is giving the expected nature of the time domain plot. 

![add_fseries_frequency](https://github.com/user-attachments/assets/7bfe3fda-0f20-4dea-91cd-0a8ba8a52217) - this is the frequency domain waveform after the change.

![add_fseries_timedomain](https://github.com/user-attachments/assets/518ceac0-209a-41ac-b09d-13cae6d41702)
- this is the expected time domain plot which we are getting after doing the above mentioned change.